doc/source/working_with_images/grub_authorization.rst

```diff
@@ -0,0 +1,70 @@
+.. _grub_authorization:
+
+Authorized access to boot menu entries
+======================================
+
+.. sidebar:: Abstract
+
+    This page provides information how to configure the GRUB boot laoder
+    to allow authorized access to its boot menu entries
+
+
+You can configure &grub; to allow access to boot menu entries depending
+on the level of authorization. You can configure multiple user accounts
+protected with passwords and assign them access to different menu entries.
+To configure authorization in GRUB;, follow these steps:
+
+1. Create and encrypt one password for each user account you want to use in
+   GRUB. Use the `grub2-mkpasswd-pbkdf2`:
+
+   .. code:: bash
+
+      $ sudo grub2-mkpasswd-pbkdf2
+      Password: ****
+      Reenter password: ****
+      PBKDF2 hash of your password is grub.pbkdf2.sha512.10000.9CA4611006F77A...
+
+2. Delete the content of the `/etc/grub.d/10_linux` file and save it.
```

Comment:
Ok, from a builder perspective you can't delete something that is not already generated. So the strategy should be to provide a KIWI project config that from a fresh build it produces an image with customized grub2.

So in KIWI terms this step should be part of `config.sh` script that is executed in chroot after installing all the packages (so in that stage there should already be a `/etc/grub.d/10_linux` file). Moreover, why do we need to save it? If this script is only the generator of the default menu entries we don't want we can simply remove it, don't we?